x-pack/plugin/core/template-resources/src/main/resources/elastic-connectors-sync-jobs.json

```diff
@@ -21,9 +25,11 @@
         "connector": {
           "properties": {
             "configuration": {
+              "dynamic": "false",
```

Comment:
Why do we need to define `dynamic` more than at the base document level?

Reply:
At the base document level the mapping is `strict`. The field `configuration` needs to accept dynamic fields because among the 20-or-so connectors we have, they all have different fields for this option. It would be a significant level of effort to define every possible mapping that can occur here. So instead, I think these fields that expect connector-dependent dynamic content should have `dynamic: false`.

Doing this allows the base-level of the doc to still have a strict policy while allowing select fields to accept dynamic sub-fields.

Reply:
Thanks for the explanation. I'll defer to your team on that.

Reply:
Team decided that top-level `strict` with some sub-fields having `false` as an exception is okay.